### ISSUE TITLE
[Bug]: createAsset mutation fails when tags id specified

### ISSUE THREAD

**Issue by natted** [pimcore/data-hub#807]
### Expected behavior

Using createAsset mutation to add image assets including tag identifier, the asset will be created successfully.

### Actual behavior

Using createAsset mutation fails with the following error when a tag id is specified with the asset.

```
{
  "errors": [
    {
      "message": "Internal server error",
      "locations": [
        {
          "line": 2,
          "column": 3
        }
      ],
      "path": [
        "createAsset"
      ],
      "extensions": {
        "debugMessage": "Pimcore\\Model\\Element\\Tag::getById(): Argument #1 ($id) must be of type int, string given, called in /var/www/html/vendor/pimcore/data-hub/src/GraphQL/Traits/ElementTagTrait.php on line 74",
        "file": "/var/www/html/vendor/pimcore/pimcore/models/Element/Tag.php",
        "line": 76,
        "trace": [
          {
            "file": "/var/www/html/vendor/pimcore/data-hub/src/GraphQL/Traits/ElementTagTrait.php",
            "line": 74,
            "call": "Pimcore\\Model\\Element\\Tag::getById('51')"
          },
          {
            "file": "/var/www/html/vendor/pimcore/data-hub/src/GraphQL/Mutation/MutationType.php",
            "line": 1007,
            "call": "Pimcore\\Bundle\\DataHubBundle\\GraphQL\\Mutation\\MutationType::getTagsFromInput(array(1))"
          },
          {
            "file": "/var/www/html/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
[SNIPPED]
```

### Steps to reproduce

Use the graphQL explorer / playground to test with a createAsset mutation similar to below 

```
mutation {
  createAsset(
    parentId: 12,
    filename: "12345_image_name.jpg",
    type: "image",
    path: "my-assets",
    input: {
      data: "aaaaaaa-your-image-data"
       metadata: [
        { name: "Title", type: "input", data: "YOUR IMAGE NAME" },
        { name: "Category", type: "input", data: "Cars" },
        { name: "Segment", type: "input", data: "Fast Cars" },
        { name: "Image Source", type: "input", data: "F360" },
        { name: "License Rights", type: "input", data: "Open" },
        { name: "Description", type: "input", data: "Image Description" },
        { name: "Image Alt", type: "input", data: "" }
      ],
      tags:[
        { id: 51 }
      ]
      
    }) {
    success
    message
  }
}
```

**Comment by natted** [comment]
The issue only occurs when tag `id` is specified (appears due to strict typing). 
If we supply tag as `path` instead of id, createAsset works. 

As a workaround - I updated line 74 within ElementTagTrait.php as follows:
Not sure if it's the right solution but can add PR if useful

```
  $tagId = intval($tag_input['id']);
  $tag = Tag::getById($tagId);
```

**Comment by mattamon** [comment]
> The issue only occurs when tag `id` is specified (appears due to strict typing). If we supply tag as `path` instead of id, createAsset works.
> 
> As a workaround - I updated line 74 within ElementTagTrait.php as follows: Not sure if it's the right solution but can add PR if useful
> 
> ```
>   $tagId = intval($tag_input['id']);
>   $tag = Tag::getById($tagId);
> ```

Hey @natted,

thank you for raising an issue for this. We would be happy about a PR. 
A simple cast should be enough though. No need for intval since we do not need a different base.

```php
  $tag = Tag::getById((int)$tag_input['id']);
```

Thank you and cheers!

**Comment by dvesh3** [comment]
@kingjia90 should we close this as the linked PR is merged?

**Comment by kingjia90** [comment]
Fixed by #808
